examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p>Can you change this <code>This notebook is created using the latest stable merlin-tensorflow-training container. </code> to "This notebook is created using the latest stable <a href="https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-tensorflow-training/tags" target="_blank">merlin-tensorflow</a>:22.XX container. "</p><p> </p><p>change this <code>We deployed the pipeline as an ensemble and ..</code> to <code>We deploy the pipeline as an ensemble and ..</code> </p><p> </p><p>Can you change the name <code> detailed example</code> to exact name of the notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks, I added changed to merlin-tensorflow instead of using merlin-tensorflow-training. Do we need to use `22.xx`. The latest stable version indicates it.

Reply:
as u like.